### PR TITLE
Remove custom responders

### PR DESCRIPTION
**Pull request by omnistegan**
This PR removes custom responders, in favor of placing them in an extension.

I'm currently hosting the extension on my github [here](https://github.com/omnistegan/solidus_responders).

This resolves the first part of https://github.com/solidusio/solidus/issues/1869

### CODE REVIEW

**Comment by mamhoff** [comment]
Let's merge this once the extension has been moved to the `solidusio-contrib` organization and a Rubygem has been cut.

**Comment by cbrunsdon** [comment]
@mamhoff I don't think we need to push to solidus-contrib if we push it to rubygems. if anyone ever runs across that extension, its not like we want them to use it.

**Comment by mamhoff** [comment]
@cbrunsdon Sure, as long as people can use the extension from their Gemfile somehow, this is good to go.

**Comment by jhawthorn** [comment]
@cbrunsdon @mamhoff I think it would be best if this was on solidus-contrib, I would like to see the tests run on TravisCI

**Comment by mamhoff** [comment]
Yeah, I like seeing tests run, too. @cbrunsdon Would you be happy with a note saying: "This gem is deprecated, don't use"?

File: CHANGELOG.md [hunk 1]
```diff
@@ -78,6 +78,8 @@
 - Emit Spree.url JS deprecation warning in all environments [\#2017](https://github.com/solidusio/solidus/pull/2017) ([jhawthorn](https://github.com/jhawthorn))
 
 
+- Customized responders have been removed. They are available in the `solidus_responders` extension
```
Comment (jhawthorn) [inline CHANGELOG.md]: Could we move this to the 2.4.0 section

**Comment by jhawthorn** [comment]
https://github.com/solidusio-contrib/solidus_responders

We've moved the repo to solidus-contrib and added a bold note at the top

**Pull request merged**